src/components/UserInfo.js

```diff
@@ -12,8 +14,13 @@ export default class UserInfo {
     return info;
   }
 
-  setUserInfo(newName, newJob) {
-    this._profileTitle.textContent = newName;
-    this._profileDescription.textContent = newJob;
+  setUserInfo(data) {
+    this._profileTitle.textContent = data.name;
+    this._profileDescription.textContent = data.about;
+  }
+
+  setAvatar(newAvatar) {
```

Comment:
It's great to have a separate setter for the avatar, since it's controlled by a different form